components/Splash/style.ts

```diff
@@ -0,0 +1,16 @@
+import { StyleSheet, Dimensions } from "react-native";
+
+const screenWidth = Dimensions.get("window").width;
+const screenHeight = Dimensions.get("window").height;
+
+export const styles = StyleSheet.create({
```

Comment:
Interesting that you choose to separate out styles. Any particular reason why?

Reply:
- I prefer keeping styles separate from logic.
- Clean code, more flexibility, easy to debug and test.
- Moreover, if styles needs to be tested, then it can be done with different test file (styles.test.tsx)